server/routes/v1/post.ts

```diff
@@ -4,12 +4,12 @@ import PostController from '../../controllers/v1/post';
 import {getAuthUser} from '../../middleware/auth';
 
 const postRouter = express.Router();
-const postController = new PostController(db.User);
+const postController = new PostController(db.Post);
 
 postRouter.get('', async (req: Request, res: Response) => {
   const limit = req.body.limit;
   const offset = req.body.offset;
-  if (!limit || !offset) {
+  if (!limit || offset == undefined) {
```

Comment:
Limit can also technically be 0 which would (correctly) return an empty array. 

I'll leave it up to you if you want to display an error for a 0 limit, or just allow it to return an empty array.

Reply:
Limit of 0 doesn't really make sense... We can handle the errors to return empty on the frontend